mobile/src/components/FormField.js

```diff
@@ -45,31 +46,45 @@ export default class FormField extends Component {
       ...attr,
       ...dynamicAttr,
     };
-    const style = attributes.editable === false
-      ? { ...attributes.style, backgroundColor: '#dddddd' }
-      : { ...attributes.style };
 
+    const editableStyle = (
+      attributes.editable === false || attributes.editable === false
```

Comment:
why attributes.editable is checked twice?

Reply:
i missed this, i am fixing it